content/en/synthetics/settings/_index.md

```diff
@@ -66,6 +69,24 @@ Note: Variable values are updated whenever the test they are extracted from runs
 [3]: https://restfulapi.net/json-jsonpath/
 {{% /tab %}}
 
+{{% tab "MFA Token" %}}  
+ 
+In the **Global Variables** tab of your **Settings** page, click **Create Global Variable** and enter a secret key or upload a QR code from your authentication provider:
```

Comment:
The first part of the sentence is a repetition of what's in the above section - let's adjust accordingly.